unittest/src/main/java/score/ServiceManagerImpl.java

```diff
@@ -244,6 +244,15 @@ public void invoke(Account from, BigInteger value, Address targetAddress, String
         }
     }
 
+    @Override
+    public Object call(Account from, BigInteger value, Address targetAddress, String method, Object... params) {
+        if (from == null) {
+            throw new NullPointerException("NullPointerException(from is null)");
```

Comment:
I think the message is duplicated. How about simply putting as follow?
```suggestion
            throw new NullPointerException("from is null");
```